### ISSUE TITLE
Every input should have a related label for tag

### ISSUE THREAD

**Issue by zufrieden** [swiss/styleguide#201]
Problem when multiple input like checkbox or radio button inline (due to bootstrap structure)
![screenshot 2014-08-08 15 51 56](https://cloud.githubusercontent.com/assets/451236/3857427/3b17bcca-1f03-11e4-972d-b976d112f870.png)
